design/FY2017/NFP-SingleDuctAirTerminalConstantVolumeNoReheat.md

```diff
@@ -0,0 +1,212 @@
+
+  A3,  \field Air Outlet Node Name
+
+       \required-field
+       \type node
+       \note This is an air outlet node from the air distribution unit. This node name should be one of the 
+       \note supply air inlet node names of a zone served by this component.
+
+  A4,  \field Air Inlet Node Name
+
+       \required-field
+       \type node
+       \note The air-inlet node name that connects the air splitter to the individual zone air distribution 
+       \note unit. This node should also be one of the outlet air node of an AirLoopHVAC:ZoneSplitter component.
+
+  N1,  \field Maximum Air Flow Rate
+
+       \required-field
+       \units m3/s
+       \minimum 0.0
+       \autosizable
+
+   A5, \field Design Specification Outdoor Air Object Name
+
+       \type object-list
+       \object-list DesignSpecificationOutdoorAirNames
+       \note When the name of a DesignSpecification:OutdoorAir object is entered, the terminal
+       \note unit will adjust flow to meet this outdoor air requirement and no more. Load is still "Uncontrolled."
+       \note If Outdoor Air Flow per Person is non-zero, then the outdoor air requirement will
+       \note be computed based on the current number of occupants in the zone, as for demand controlled ventilation.
+       \note At no time will the supply air flow rate exceed the value for Maximum Air Flow Rate.
+       \note If this field is blank, then the terminal unit will not be controlled for outdoor air flow.
+
+   A6; \field Per Person Ventilation Rate Mode
+
+       \type choice
+       \key CurrentOccupancy
+       \key DesignOccupancy
+       \default CurrentOccupancy
+       \note CurrentOccupancy models demand controlled ventilation using the current number of people
+       \note DesignOccupancy uses the total Number of People in the zone and is constant
+
+```
+
+## Input Description ##
+
+### Inputs
+
+#### Field: Name
+
+A unique user assigned name for each AirTerminal:SingleDuct:ConstantVolume:NoReheat component. Any other component that needs to use this terminal air component will refer to it by this name.
+
+#### Field: Availability Schedule Name
+
+The name of the schedule (ref: Schedule) that denotes whether the component can run during a given time period. A schedule value greater than 0 (usually 1 is used) indicates that the component can be on during the time period. A value less than or equal to 0 (usually 0 is used) denotes that the component must be off for the time period. If this field is blank, the schedule has values of 1 for all time periods.
+
+#### Field: Air Outlet Node Name
+
+This is an air outlet node from the air distribution unit. This node name should be one of the supply air inlet node names of a zone served by this component.
+
+#### Field: Air Inlet Node Name
+
+The air-inlet node name that connects the air splitter to the individual zone air distribution unit. This node should also be one of the outlet air node of an AirLoopHVAC:ZoneSplitter component.
```

Comment:
air-inlet --> air inlet
one of the outlet air node . . .  --> . . . nodes of an AirLoopHVAC:ZoneSplitter or AirLoopHVAC:SupplyPlenum component. (this same change is probably needed in all other air terminal units?)

Reply:
I will add the notes to the other air terminal objects as well.